### PR TITLE
5 feature 5 player factories

### PR DESCRIPTION
**Pull request by meiiph**
Add the necessary player factories to better facilitate creating the players once the game is started. Persistence for stats and checking if a player name has already been taken when creating a player have yet to be implemented.

### CODE REVIEW

File: src/main/java/entities/ComputerPlayerFactory.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package entities;
+
```
Comment (SMateen21) [inline src/main/java/entities/ComputerPlayerFactory.java:2]: Perhaps you could add some Javadocs to this class just briefly describing what it does.

**Review by SMateen21 — COMMENTED** [review]
Looks great. It's a good use of factory design pattern. You could add some Javadocs for the class just to have more complete documentation. but looks good overall.

File: src/main/java/entities/ComputerPlayerFactory.java (hunk 1 above)
Comment (meiiph) [reply src/main/java/entities/ComputerPlayerFactory.java:2]: Sounds good, I'll add Javadocs to all the classes.

**Review by SMateen21 — APPROVED** [review]
All changes done, looks good to me.

**Review by lilyxmeng — APPROVED** [review]
Looks good

**Pull request merged**